website/content/docs/v0.3/Tutorial/create-workload.md

```diff
@@ -0,0 +1,91 @@
+---
+description: "A guide for bootstrapping Sidero management plane"
+weight: 1
+---
+
+# Create a Workload Cluster
+
+Once created and accepted, you should see the servers that make up your ServerClasses appear as "available":
+
+```bash
+$ kubectl get serverclass
+NAME      AVAILABLE                                  IN USE
+any       ["00000000-0000-0000-0000-d05099d33360"]   []
+```
+
+## Generate Cluster Manifests
+
+We are now ready to generate the configuration manifest templates for our first workload
+cluster.
+
+There are several configuration parameters that should be set in order for the templating to work properly:
+
+- `CONTROL_PLANE_ENDPOINT`: The endpoint used for the Kubernetes API server (e.g. `https://1.2.3.4:6443`).
+  This is the equivalent of the `endpoint` you would specify in `talosctl gen config`.
+  There are a variety of ways to configure a control plane endpoint.
+  Some common ways for an HA setup are to use DNS, a load balancer, or BGP.
+  A simpler method is to use the IP of a single node.
+  This has the disadvantage of being a single point of failure, but it can be a simple way to get running.
+- `CONTROL_PLANE_SERVERCLASS`: The server class to use for control plane nodes.
+- `WORKER_SERVERCLASS`: The server class to use for worker nodes.
+- `KUBERNETES_VERSION`: The version of Kubernetes to deploy (e.g. `v1.21.1`).
+- `CONTROL_PLANE_PORT`: The port used for the Kubernetes API server (port 6443)
+
+For instance:
+
+```bash
+export CONTROL_PLANE_SERVERCLASS=any
+export WORKER_SERVERCLASS=any
+export KUBERNETES_VERSION=v1.21.1
+export CONTROL_PLANE_PORT=6443
+export CONTROL_PLANE_ENDPOINT=1.2.3.4
+
+clusterctl config cluster cluster-0 -i sidero > cluster-0.yaml
+```
+
+Take a look at this new `cluster-0.yaml` manifest and made any changes as you
+see fit.
+Feel free to adjust the `replicas` field of the `TalosControlPlane` and `MachineDeployment` objects to match the number of machines you want in your controlplane and worker sets, respecively.
+`MachineDeployment` (worker) count is allowed to be 0.
+
+Of course, these may also be scaled up or down _after_ they have been created,
+as well.
+
+When you are satisfied with your configuration, go ahead and apply it to Sidero:
+
+```bash
+kubectl apply -f cluster-0.yaml
+```
+
+## Retrieve the Talosconfig
```

Comment:
should there be some section which explains how to wait for the cluster to be fully provisioned?